bm-persona/Dining/DiningDetailViewController.swift

```diff
@@ -0,0 +1,504 @@
+//
+//  DiningDetailViewController.swift
+//  bm-persona
+//
+//  Created by Shawn Huang on 4/4/20.
+//  Copyright © 2020 RJ Pimentel. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+fileprivate let kCardPadding: UIEdgeInsets = UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16)
+fileprivate let kViewMargin: CGFloat = 16
+
+class DiningDetailViewController: UIViewController {
+    
+    var diningHall: DiningHall!
+    var locationManager = CLLocationManager()
+    var location: CLLocation?
+    var control: TabBarControl!
+    var meals: MealMap!
+    var mealNames: [MealType]!
+    var menuView: FilterTableView = FilterTableView<DiningItem>(frame: .zero, filters: [])
+    static let cellHeight: CGFloat = 45
+    static let cellSpacingHeight: CGFloat = 5
+    static let mealTimesChronological = ["breakfast": 0, "brunch": 1, "lunch": 2, "dinner": 3, "late night": 4, "other": 5]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .white
+        
+        locationManager.delegate = self
+        setUpOverviewCard()
+        setupMenuControl()
+        setupMenu()
+        
+        if CLLocationManager.locationServicesEnabled() {
+            locationManager.requestLocation()
+        }
+    }
+    
+    let nameLabel: UILabel = {
+        let label = UILabel()
+        label.font = Font.semibold(24)
```

Comment:
Ah, sorry for forgetting to mention this, but could you make this label bold instead of semibold?